packages/aws-cdk/lib/api/util/sdk_ini_file.ts

```diff
@@ -44,9 +45,9 @@ export class SharedIniFile {
 
   private ensureFileLoaded() {
     if (!this.parsedContents) {
-      this.parsedContents = (AWS as any).util.ini.parse(
-        (AWS as any).util.readFileSync(this.filename)
-      );
+      this.parsedContents = fs.pathExistsSync(this.filename)
```

Comment:
Are you sure this is necessary? I'm positive I tested this WITHOUT an `~/.aws/config` file and no crash.

Reply:
My observations show that a missing `~/.aws/credentials` would blow up right this spot.